### PR TITLE
hatchback+suv

### PR DESCRIPTION
**Pull request by AlisGraveNil**
## Description

Created .json files for both keyboards

### CODE REVIEW

**Comment by AlisGraveNil** [comment]
I don't remember the instructions for how to get the CLI to spit out the commit number

File: public/keymaps/c/clawsome_hatchback_default.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    [
+      "KC_ESC",             "KC_F1",   "KC_F2",   "KC_F3",   "KC_F4",   "KC_F5",   "KC_F6",   "KC_F7",   "KC_F8",   "KC_F9",   "KC_F10",  "KC_F11",  "KC_F12",                "KC_PSCR", "KC_SLCK", "KC_PAUS",
+      "KC_GRV",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS", "KC_EQL",  "KC_BSPC",               "KC_INS",  "KC_HOME", "KC_PGUP",
+      "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSLS",               "KC_DEL",  "KC_END",  "KC_PGDN",
+      "KC_CAPS", "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_SCLN", "KC_QUOT",            "KC_ENT",
+      "KC_LSFT",            "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH", "KC_RSFT",                                     "KC_UP",
+      "KC_LCTL", "KC_LGUI", "KC_LALT",            "KC_SPC",               "KC_SPC",           "KC_SPC",             "KC_RALT"  "KC_RGUI", "KC_MENU", "KC_RCTL",               "KC_LEFT", "KC_DOWN", "KC_RGHT"
```
Comment (noroadsleft) [inline public/keymaps/c/clawsome_hatchback_default.json:13]:
```suggestion
      "KC_LCTL", "KC_LGUI", "KC_LALT",            "KC_SPC",               "KC_SPC",           "KC_SPC",             "KC_RALT", "KC_RGUI", "KC_MENU", "KC_RCTL",               "KC_LEFT", "KC_DOWN", "KC_RGHT"
```

File: public/keymaps/c/clawsome_suv_default.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "keyboard": "clawsome/suv",
+  "keymap": "default",
+  "commit": "x",
```
Comment (noroadsleft) [inline public/keymaps/c/clawsome_suv_default.json:4]:
```suggestion
  "commit": "22cd151fc359f48d75e361bd46a25ab6b40425fa",
```

File: public/keymaps/c/clawsome_hatchback_default.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "keyboard": "clawsome/hatchback",
+  "keymap": "default",
+  "commit": "ed7e61fa083e19e2716f7ae37520c3ca9db74ae8",
+  "layout": "LAYOUT_all",
```
Comment (noroadsleft) [inline public/keymaps/c/clawsome_hatchback_default.json:5]:
```suggestion
  "layout": "LAYOUT",
```

File: public/keymaps/c/clawsome_suv_default.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  "layout": "LAYOUT_all",
```
Comment (noroadsleft) [inline public/keymaps/c/clawsome_suv_default.json:5]:
```suggestion
  "layout": "LAYOUT",
```

File: public/keymaps/c/clawsome_suv_default.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+      "KC_GRV",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS", "KC_EQL",  "KC_BSPC",    "KC_INS",  "KC_HOME", "KC_PGUP",  "KC_NLCK", "KC_PSLS", "KC_PAST", "KC_PMNS",
+      "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSLS",    "KC_DEL",  "KC_END",  "KC_PGDN",  "KC_P7",   "KC_P8",   "KC_P9",   "KC_PPLS",
+      "KC_CAPS", "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_SCLN", "KC_QUOT",            "KC_ENT",                                       "KC_P4",   "KC_P5",   "KC_P6",
+      "KC_LSFT",            "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH", "KC_RSFT",                          "KC_UP",               "KC_P1",   "KC_P2",   "KC_P3",   "KC_PENT",
+      "KC_LCTL", "KC_LGUI", "KC_LALT",            "KC_SPC",               "KC_SPC",           "KC_SPC",             "KC_RALT"  "KC_RGUI", "KC_MENU", "KC_RCTL",    "KC_LEFT", "KC_DOWN", "KC_RGHT".  "KC_P0",              "KC_PDOT",
```
Comment (noroadsleft) [inline public/keymaps/c/clawsome_suv_default.json:13]:
```suggestion
      "KC_LCTL", "KC_LGUI", "KC_LALT",            "KC_SPC",               "KC_SPC",           "KC_SPC",             "KC_RALT"  "KC_RGUI", "KC_MENU", "KC_RCTL",    "KC_LEFT", "KC_DOWN", "KC_RGHT",  "KC_P0",              "KC_PDOT",
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: public/keymaps/c/clawsome_suv_default.json [hunk 6]
```diff
@@ -1,16 +1,16 @@
...
       "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSLS",    "KC_DEL",  "KC_END",  "KC_PGDN",  "KC_P7",   "KC_P8",   "KC_P9",   "KC_PPLS",
       "KC_CAPS", "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_SCLN", "KC_QUOT",            "KC_ENT",                                       "KC_P4",   "KC_P5",   "KC_P6",
       "KC_LSFT",            "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH", "KC_RSFT",                          "KC_UP",               "KC_P1",   "KC_P2",   "KC_P3",   "KC_PENT",
-      "KC_LCTL", "KC_LGUI", "KC_LALT",            "KC_SPC",               "KC_SPC",           "KC_SPC",             "KC_RALT"  "KC_RGUI", "KC_MENU", "KC_RCTL",    "KC_LEFT", "KC_DOWN", "KC_RGHT".  "KC_P0",              "KC_PDOT",
+      "KC_LCTL", "KC_LGUI", "KC_LALT",            "KC_SPC",               "KC_SPC",           "KC_SPC",             "KC_RALT"  "KC_RGUI", "KC_MENU", "KC_RCTL",    "KC_LEFT", "KC_DOWN", "KC_RGHT",  "KC_P0",              "KC_PDOT",
```
Comment (noroadsleft) [inline public/keymaps/c/clawsome_suv_default.json:13]: I missed this on the previous review.

```suggestion
      "KC_LCTL", "KC_LGUI", "KC_LALT",            "KC_SPC",               "KC_SPC",           "KC_SPC",             "KC_RALT", "KC_RGUI", "KC_MENU", "KC_RCTL",    "KC_LEFT", "KC_DOWN", "KC_RGHT",  "KC_P0",              "KC_PDOT"
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**